### ISSUE TITLE
NEED THE OREO SOURCE OF MIA1

### ISSUE THREAD

**Issue by aalakib07** [MiCode/Xiaomi_Kernel_OpenSource#843]
:D

**Comment by muralivijay** [comment]
Rebase it nought kernel to oreo

**Comment by dragonpt** [comment]
Will be up on March

**Comment by bgcngm** [comment]
@dragonpt what are you doing here? Long time no see!

**Comment by muralivijay** [comment]
@dragonpt Xiaomi does not release . New android kernel source. They will only release base firmware version . Like if a Xiaomi device launched with marshmallow . They will only release marshmallow kernel  . Also look other Xiaomi device land, mido, Kenzo, markw stil having marshmallow source . But how they get Oreo source. You need rebase nougat to Oreo using CAF source . I hope u understand this 

Also my mi A1 Oreo boots with rebase source

**Comment by dragonpt** [comment]
There will be further releases, has been an busy week to what concerns to XI open source 
( as you can see by yourself)

Things will be more straight forward now, hopefully

@bgcngm nice to see you here, didn't knew you worked in /have a Xiaomi ;)

**Comment by bgcngm** [comment]
@dragonpt MTK is past. :) Hit me up on hangouts.

**Comment by 0pLuS0** [comment]
STOP SPAMMING PEOPLE AND READ! OR NEXT TIME YOU MIGHT BE BANNED!

https://github.com/MiCode/Xiaomi_Kernel_OpenSource/issues/854

**Comment by m-p-3** [comment]
It's out
https://github.com/MiCode/Xiaomi_Kernel_OpenSource/commit/65d05bc7ed777f448743a1d41a9e737858603bc6

**Comment by dragonpt** [comment]
Yes of course its released ;)

Cheera

**Comment by namoen0301** [comment]
Mi A1 is Android One, Please contact Google, not Xiaomi

**Comment by scafroglia93** [comment]
Ahahahahaha

**Comment by scafroglia93** [comment]
Ciao MAMMA!

**Comment by namoen0301** [comment]
Let me say it again: 
> **Mi A1 is Android One**

Why are you still ask xiaomi to release kernel source?
I've said as much as I'm prepared to say at this point.

**Comment by scafroglia93** [comment]
Android One devices are not managed by Google but by the device manufacturer (xiaomi)

**Comment by namoen0301** [comment]
@scafroglia93
Yes, Manufacturer is Xiaomi, But Mi A1 not running MIUI but Pure Android, You just ask Xiaomi to release Kernel Source, But Can you really be certain it works?
By the way, Mi A1 bootloader unlock is different from other Xiaomi device.

**Comment by scafroglia93** [comment]
What's the problem ?

The sources are always managed by the manufacturer, it has nothing to do with the system
If you buy cheap junk it's not my fault !

**Comment by namoen0301** [comment]
Do as you please. I use OnePlus 6, Redmi Note 5 and iPhone X.

I will never reply this issue.

**Comment by muralivijay** [comment]
Lol close this topic kernel source already released. So many spams coming

**Comment by ryan-andri** [comment]
@scafroglia93 agreed.

**Comment by dragonpt** [comment]
Sorry but you all seem just spoiled kids.

If XI doesn't release kernel, they are our archenemies
If they do, and just it happens to be a bit old, archenemies they are still

This ain't XDA or something

Does the released kernel source boots, and all the hardware is functional? Yes? Then?

As long as the released source match's the release , that's fine, and that's what is expected .

Now don't expected XI , or as matter of fact any OEM,to keep updating their public source on daily basis.

Just be thankful of what you have

PS: what are some of you talking about?
     The source is already released ...

**Comment by scafroglia93** [comment]
Then explain to me why the phone manufacturer I'm currently using updates the CAF releases ?

**Comment by dragonpt** [comment]
How many OTAs on the A1, has been released?

The kernel source released, match's the PR
, if there's any OTA afterwards with kernel changes, then yes, kernel source should be updated to match that release

**Comment by YaroST12** [comment]
@scafroglia93 probably because its userbase is 1/1000 of Xiaomi? If Xiaomi releases a bugged update - it's getting mentioned on all Android news sites/forums. If BQ releases a bugged update... Doubt anyone will even notice.
